modules/ipi-install-validation-checklist-for-nodes.adoc

```diff
@@ -8,19 +8,21 @@
 
 .When using the `provisioning` network
 
-* [ ] NIC1 VLAN is configured for the `provisioning` network. (optional)
-* [ ] NIC1 is PXE-enabled on the provisioner, control plane (master), and worker nodes when using a `provisioning` network. (optional)
-* [ ] NIC2 VLAN is configured for the `baremetal` network.
+* [ ] (Optional) NIC VLAN is configured for the `provisioning` network.
```

Comment:
These aren't optional if you're using the provisioning network. The optional part is handled by the next section without a provisioning network.